lib/bindings/langs/flutter/justfile

```
@@ -54,6 +55,16 @@ build-android profile='frb-min':
 build-other profile='frb-min':
 	dart scripts/build_other.dart --profile {{profile}}
 
+build-bindings:
```

Can we rename 'bindings' to 'uniffi' in all occurrences?
```suggestion
build-bindings:
```

---

Done [00bfede](https://github.com/breez/breez-sdk-liquid/pull/488/commits/00bfede5f84fbd40e966b444db74a14a88fa0695)